modules/developer_manual/pages/app/tutorial/development_environment.adoc

```diff
@@ -20,10 +20,24 @@ In addition to these, there are several additional, commonly used,
 directories:
 
 * `css/`: Contains the CSS files
+* `img/`: Contains the image files
 * `js/`: Contains the JavaScript files
 * `templates/`: Contains the templates
 * `tests/`: Contains the tests
```

Comment:
should we also add the "vendor" folder ?
usually there's also a "lib" folder with the classes

Reply:
I've done a scan and not found any vendor directories yet. However, I did find `bin` and `l10n` directories using [a bash one-liner](https://gist.github.com/settermjd/5c15e1d1683d3f75d077349adaee9718), so have included those.